File: browsing-service/src/main/java/com/rip/browsing_service/dto/PlenaryProtocolDto.java
package com.rip.browsing_service.dto;

import java.util.List;

public record PlenaryProtocolDto(
    String id,
    String date,
    String title,
    String summary,
    int speakerCount,
    int totalWords,
    List<PlenaryProtocolPartyStatsDto> partyStats
) {}

File: client/src/types/PlenaryProtocolDto.ts
import type { PlenaryProtocolPartyStatsDto } from "./PlenaryProtocolPartyStatsDto";

export interface PlenaryProtocolDto {
  id: string;
  date: string | null;
  title: string;
  summary: string | null;
  speakerCount: number;
  totalWords: number;
  partyStats: PlenaryProtocolPartyStatsDto[];
}
